src/ralph/cli.py

```diff
@@ -393,3 +397,52 @@ def list_(details, new, backend, **options):
 
     if counter == 0:
         logger.warning("Configured %s backend contains no archive", backend)
+
+
+@cli.command()
+@click.option(
+    "-h",
+    "--host",
+    type=str,
+    default=DEFAULT_RUNSERVER_HOST,
+    help=(
+        "Serve the LRS-compatible API on this host in the container "
+        f"(default {DEFAULT_RUNSERVER_HOST})"
+    ),
+)
+@click.option(
+    "-p",
+    "--port",
+    type=int,
+    default=DEFAULT_RUNSERVER_PORT,
+    help=f"Serve the LRS-compatible API on this port (default {DEFAULT_RUNSERVER_PORT})",
+)
+@click.option(
+    "-f",
+    "--auth-file",
+    type=str,
+    default="credentials/users.json",
+    help="File containing authentification & authorization information for API calls",
+)
+def runserver(host, port, auth_file):
```

Comment:
IMO making this path configurable using the environment or a config file with a relevant default is sufficient (using the `config` utility).

```suggestion
def runserver(host, port):
```